tests/providers/amazon/aws/operators/test_eks.py

```diff
@@ -92,8 +91,8 @@ class CreateNodegroupParams(TypedDict):
     nodegroup_role_arn: str
 
 
-class TestEksCreateClusterOperator(unittest.TestCase):
-    def setUp(self) -> None:
+class TestEksCreateClusterOperator:
```

Comment:
Also I remember that I have an issue with configurations mutability between tests